utils/Enum.js

```diff
@@ -0,0 +1,12 @@
+class Enum {
```

Comment:
First of all, it should tell what is happening to the developer, not user :P
And I think the developer knows what `Enum` means.
I freeze object to avoid overwrites because I wanted to achieve variable with a not changeable dictionary.